src/components/Layout.js

```diff
@@ -85,4 +85,14 @@ exports.addSystemTestSupport = function(){
         browser.get(url);
         browser.sleep(3000);
     });
+};
+
+exports.acceptAlert = function(){
+    browser.sleep(1000);
+    if(browser.browserName === 'chrome'){
+        browser.switchTo().alert().accept();
+    }
+    if(browser.browserName === 'phantomjs'){
```

Comment:
in such case I would use 'else if' cause if the browser is chrome I don't want the code to check if the browser is phantom